README.md

```diff
@@ -1256,6 +1256,674 @@ The type of cost
+implement that:
+
+```python
+objective_functions = ObjectiveList()
+if objective_name == "MINIMIZE_PREDICTED_COM_HEIGHT":
+    objective_functions.add(ObjectiveFcn.Mayer.MINIMIZE_PREDICTED_COM_HEIGHT, weight=-1)
+elif objective_name == "MINIMIZE_COM_POSITION":
+    objective_functions.add(ObjectiveFcn.Lagrange.MINIMIZE_COM_POSITION, axis=Axis.Z, weight=-1)
+elif objective_name == "MINIMIZE_COM_VELOCITY":
+    objective_functions.add(ObjectiveFcn.Lagrange.MINIMIZE_COM_VELOCITY, axis=Axis.Z, weight=-1)
+```
+
+Another interesting point of this example is the definition of the constraints. Thanks to the `com_constraints` boolean, 
+the user can easily choose to apply constraints on the center of mass. Here is the definition of the constraints for our 
+example:
+
+```python
+constraints = ConstraintList()
+if com_constraints:
+    constraints.add(
+        ConstraintFcn.TRACK_COM_VELOCITY,
+        node=Node.ALL,
+        min_bound=np.array([-100, -100, -100]),
+        max_bound=np.array([100, 100, 100]),
+    )
+    constraints.add(
+        ConstraintFcn.TRACK_COM_POSITION,
+        node=Node.ALL,
+        min_bound=np.array([-1, -1, -1]),
+        max_bound=np.array([1, 1, 1]),
+    )
+```
+
+This example is designed to show how to use `min_bound` and `max_bound` values so they define inequality constraints 
+instead of equality constraints, which can be used with any `ConstraintFcn`. This example is closed to the 
+example_inequality_constraint.py file you can find in 'examples/getting_started/example_inequality_constraint.py'.
+
+### The spring_load.py file 
+This trivial spring example targets to have the highest upward velocity. It is however only able to load a spring by
+pulling downward and afterward to let it go so it gains velocity. It is designed to show how one can use the external
+forces to interact with the body.
+
+This example is closed to the custom_dynamics.py file you can find in 'examples/getting_started/custom_dynamics.py'. 
+Indeed, we generate an external force thanks to the custom_dynamic function. Then, we configure the dynamics with 
+the `custom_configure` function. 
+
+### The track_markers_2D_pendulum.py file
+
+This example uses the data from the balanced pendulum example to generate the data to track.
+When it optimizes the program, contrary to the vanilla pendulum, it tracks the values instead of 'knowing' that
+it is supposed to balance the pendulum. It is designed to show how to track marker and kinematic data.
+
+Note that the final node is not tracked. 
+
+In this example, we use both `ObjectiveFcn.Lagrange.TRACK_MARKERS` and `ObjectiveFcn.Lagrange.TRACK_TORQUE` objective 
+functions to track data, as shown in the definition of the objective functions used in this example:
+
+```python
+objective_functions = ObjectiveList()
+objective_functions.add(
+    ObjectiveFcn.Lagrange.TRACK_MARKERS, axis_to_track=[Axis.Y, Axis.Z], weight=100, target=markers_ref
```

Comment:
Add the use of axis_to_track in the description below

Reply:
Done